src/tmx/TmxUtils/src/kafka/kafka_client.h

```diff
@@ -0,0 +1,22 @@
+#ifndef KAFKA_CLIENT_H
```

Comment:
The plan is eventually to make this a separate library that both CARMA-Streets and V2X-Hub can include as a dependency but currently CARMA-Streets and V2X-Hub use different logging libraries so that implementation is slightly different.

Reply:
Yeah I think that is something we can evaluate separately from this PR. We can add a github issue for it